pegasus/sites.v3/code.org/views/unsupported_browser.haml

```diff
@@ -5,7 +5,15 @@
 
 :javascript
   $(document).ready(function () {
+    if (!window.isUnsupportedBrowser) {
```

Comment:
I don't envision the current approach as a long term solution, but rather a short term solution that (a) reduces noise and (b) provides us with some additional information about what's failing (i.e. is it actually a timing issue or something else) that will hopefully make it easier for us to find a long term solution.